recipes/gdal/post_3.5.0/conanfile.py

```diff
@@ -3,50 +3,10 @@
 from conan.tools.files.files import rmdir
 from conan.tools.microsoft import is_msvc
 from conans import CMake
+import functools
 import os
 import re
```

Comment:
Linter says this is unused

Reply:
If I remove it, I get:
```
  File "/home/jpilet/invision/conan-center-index/recipes/gdal/post_3.5.0/conanfile.py", line 288, in GdalConan
    @functools.lru_cache(1)
NameError: name 'functools' is not defined
```

So that looks like a false alarm

Reply:
I think you removed the wrong import 🙈